
The first is a real fix not relying on accidently include and the later
is a fix for cross buliding due f*cked up bash compiling.

  - Rene Rebe <rene@exactcode.de>

--- bash-3.1/lib/sh/winsize.c.vanilla	2005-12-31 18:03:04.000000000 +0000
+++ bash-3.1/lib/sh/winsize.c	2005-12-31 18:03:53.000000000 +0000
@@ -29,6 +29,7 @@
 #endif
 
 #include <sys/ioctl.h>
+#include <termios.h>
 
 #if !defined (STRUCT_WINSIZE_IN_SYS_IOCTL)
 /* For struct winsize on SCO */
--- bash-3.1/config.h.in.vanilla	2005-12-31 18:01:08.000000000 +0000
+++ bash-3.1/config.h.in	2005-12-31 18:01:17.000000000 +0000
@@ -21,6 +21,8 @@
 #ifndef _CONFIG_H_
 #define _CONFIG_H_
 
+#define __u_char_defined
+
 /* Configuration feature settings controllable by autoconf. */
 
 /* Define JOB_CONTROL if your operating system supports
